### ISSUE TITLE
The project is dead or alive?

### ISSUE THREAD

**Issue by devopsjohndoe** [PHPFusion/PHPFusion#2428]
**Is your feature request related to a problem? Please describe.**
Want a community site, like Cosplay and Anime, a lot of photos and videos will be, maybe some forum on that CMS

**Describe the solution you'd like**
Happy to choose the right CMS for a site that will be live 10+ years

**Describe alternatives you've considered**
Still searching

**Additional context**
News, Articles, Forum, Photo & Video gallery is a must!

**Comment by meangczac** [comment]
Yes we are around.

**Comment by devopsjohndoe** [comment]
@FrederickChan @Der-WU @Dahlgren @kneekoo 
Seems like main site was hacked or idk

![Image](https://github.com/user-attachments/assets/79c65cda-d501-42bc-a16b-051016dc7388)

**Comment by odags-Dev** [comment]
@devopsjohndoe 
> Seems like main site was hacked or idk

No, it's not hacked, it's currently undergoing maintenance, details on [the official discord server](https://discord.gg/nuaAYbV)

**Comment by devopsjohndoe** [comment]
@ODAGS-Dev  Can you share discord, because the invite is old

**Comment by odags-Dev** [comment]
@devopsjohndoe https://discord.gg/MvzfxNRyyK

**Comment by meangczac** [comment]
I'm just having a holiday moment 😄
